### ISSUE TITLE
[Question] Slow package install from git branch

### ISSUE THREAD

**Issue by HarshRohila** [privatenumber/git-publish#6]
I am using Azure Repo
I published my package with this package on a git branch. Installing from git branch took 4mins.
But if I publish the same package on Azure Registry, install from it took 14s
Is it expected?

**Comment by privatenumber** [comment]
Yeah, I'm not completely sure why, but I think there are optimizations it misses out on when it's not on the npm registry (SSH/Git overhead, no tarball, etc).

I was observing slowness when using it with older codebases with outdated npm. But in newer codebases with pnpm, I haven't had any problems.
